### PR TITLE
CORE: Added SMN T1 Merits

### PR DESCRIPTION
**Pull request by Zenny89**
CORE: Added SMN T1 Merits

### CODE REVIEW

File: src/map/utils/petutils.cpp [hunk 1]
```diff
@@ -1307,6 +1307,15 @@ namespace petutils
                 }
             }
 
+            CCharEntity* PChar = (CCharEntity*)PMaster;
```
Comment (teschnei) [inline src/map/utils/petutils.cpp]: still need to move this into the if

**Comment by TeoTwawki** [comment]
@Zenny89 you can use the "amend" feature to edit your last commit without adding a new one (so you won't have to squash again) and then force push to update this pull request with the change teschnei pointed out.

**Comment by Zenny89** [comment]
I deleted my last repository while playing around with GitHub last week, so that's why it's a new request. I don't quite understand why/how I can move that into the if without the "If" statement making sense.

**Comment by Zenny89** [comment]
Corrected

**Pull request merged**